tests/tests.py

```diff
@@ -153,5 +154,59 @@ def test_translate_not_valid_phrase(self):
         current_result = vb.translate("asldkfj", "en", "ru")
         self.assertEqual(current_result, False)
 
+    def test_respond_as_dict_1(self):
+        data = json.loads('[{"text": "hummus", "seq": 0}]')
```

Comment:
Well these are meant to be unit test, so I should be checking that the class works in itself. Calling the API would be more or less functional testing. Ideally the api requests should be mocked since you don't control it, it's external. For example; if the api were to have an extra definition added, the test would fail and that wouldn't be as result of the class not working.